app/controllers/users_controller.rb

```diff
@@ -93,6 +93,14 @@ def impersonate
     end
   end
 
+  def invalidate_jwt
+    @user = find_resource(:edit_users)
+    User.find_by(id: @user.id).jwt_secret&.destroy
```

Comment:
I still think that 
```suggestion
    find_resource(:edit_users).jwt_secret&.destroy
```
will do the trick